test/functional/p2p_filter.py

```diff
@@ -93,6 +94,21 @@ def test_size_limits(self, filter_node):
 
         filter_node.send_and_ping(msg_filterclear())
 
+    def test_msg_mempool(self):
+        self.log.info("Check that a node with bloom filters enabled services p2p mempool messages")
+        filter_peer = FilterNode()
+
+        self.log.info("Send a tx to the node before connecting")
+        filter_address = self.nodes[0].decodescript(filter_peer.watch_script_pubkey)['addresses'][0]
+        txid = self.nodes[0].sendtoaddress(filter_address, 90)
+
+        self.log.info("Send a mempool msg after connecting and check that the tx is received")
+        self.nodes[0].add_p2p_connection(filter_peer)
+        filter_peer.send_and_ping(filter_peer.watch_filter_init)
+        self.nodes[0].p2p.send_message(msg_mempool())
+        filter_peer.wait_for_tx(txid)
+        assert filter_peer.tx_received
```

Comment:
nit in commit facd5f9765:

This assert is not needed. A stronger assert was checked in the previous line, no?